### PR TITLE
[type:feat] add projectVersion when admin and bootstrap start

### PR DESCRIPTION
**Pull request by YxYL6125**
- add projectVersion to `bannet.txt` in admin&bootstrap
- add projectVersion to `applition.yml` in admin&bootstrap
- add resouces and plugin to config in project `pom.xml`

result:
image-20231115111525295

image-20231115111541691
Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by moremind** [comment]
use ShenYu Version?

**Comment by YxYL6125** [comment]
> use ShenYu Version?

has changed

File: pom.xml [hunk 1]
```diff
@@ -665,7 +665,25 @@
...
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+            </resource>
+        </resources>
+        
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```
Comment (moremind) [inline pom.xml:677]: remove this plugin?

File: shenyu-bootstrap/src/main/resources/application.yml [hunk 2]
```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+ShenyuVersion: @project.version@
```
Comment (moremind) [inline shenyu-bootstrap/src/main/resources/application.yml:16]: use project.version not ShenYuVersion?

**Review by moremind — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (YxYL6125) [reply pom.xml:677]: has removed

File: shenyu-bootstrap/src/main/resources/application.yml (hunk 2 above)
Comment (YxYL6125) [reply shenyu-bootstrap/src/main/resources/application.yml]: has changed

**Review by moremind — CHANGES_REQUESTED** [review]

File: shenyu-bootstrap/src/main/resources/application.yml [hunk 3]
```diff
@@ -143,58 +146,58 @@ shenyu:
         mod: k8s #manul
         defaultK8sSecretNamespace: shenyu-ingress
         defaultK8sSecretName: default-cert
-#        mod: manual
```
[the remark this answers is not in the document]
Comment (YxYL6125) [reply shenyu-bootstrap/src/main/resources/application.yml]: rollbacked

**Pull request closed without merging**